packages/date-time-utilities/jest.config.js

```diff
@@ -0,0 +1,6 @@
+let path = require('path');
+let { createConfig, resolveMergeStylesSerializer } = require('@uifabric/build/jest/jest-resources');
+module.exports = createConfig({
+  setupFiles: [path.resolve(path.join(__dirname, 'config', 'tests.js'))],
+  snapshotSerializers: [resolveMergeStylesSerializer()],
+});
```

Comment:
This can be simplified as follows:
```suggestion
// @ts-check
const { createConfig } = require('@uifabric/build/jest/jest-resources');

module.exports = createConfig({});
```